common/containers/Tabs/GenerateWallet/components/Mnemonic/Mnemonic.tsx

```diff
@@ -113,7 +104,13 @@ export default class GenerateMnemonic extends React.Component<{}, State> {
     if (this.state.isConfirming) {
       this.setState({ isConfirmed: true });
     } else {
-      this.setState({ isConfirming: true });
+      const shuffledWords = shuffle([...this.state.words]);
```

Comment:
Shuffle already creates a new array, no need to `...`

Reply:
Right, forgot I moved that to the function.